mono/metadata/icall.c

```diff
@@ -8973,11 +8973,12 @@ gconstpointer
 mono_lookup_internal_call_full_with_flags (MonoMethod *method, gboolean warn_on_missing, guint32 *flags)
 {
 	char *sigstart;
```

Comment:
Null-initialize this as well?

Reply:
It is never `free`d but ok.

Reply:
The `int`s too then?

Reply:
I also rebased to master since this was old.

Reply:
Oh you're right, I'm dumb.